Assets/Scripts/View/PlayerCanvas/Cell.cs

```diff
@@ -45,21 +56,35 @@ protected override void SetUpgradeMode(bool owner)
         Upgrade.SetActive(tof);
     }
 
+    void SendMessage(byte id)
```

Comment:
👍  for creating a special function for sending the message! Maybe there is an opportunity to do the same in the back end ;)